packages/account/src/Constants/routes-config.js

```diff
@@ -25,101 +25,120 @@ import {
 // Error Routes
 const Page404 = React.lazy(() => moduleLoader(() => import(/* webpackChunkName: "404" */ 'Modules/Page404')));
 
+// route_key is used as the value for the react key prop
 // Order matters
 const initRoutesConfig = ({ is_appstore }) => [
     {
+        route_key: 1,
```

Comment:
Can we use the path or title as a key? Maintaining an index for the key is not ideal.

Reply:
@balakrishna-deriv, Is it okay to use a `/` as a `key`, because I thought that might cause problems 🤔 But I'll update it to use those regardless